shell/platform/windows/cursor_handler.cc

```diff
@@ -45,6 +51,79 @@ void CursorHandler::HandleMethodCall(
     const auto& kind = std::get<std::string>(kind_iter->second);
     delegate_->UpdateFlutterCursor(kind);
     result->Success();
+  } else if (method.compare(kSetSystemCursorMethod) == 0) {
+    const auto& arguments = std::get<EncodableMap>(*method_call.arguments());
+    auto buffer_iter =
+        arguments.find(EncodableValue(std::string(kCustomCursorBufferKey)));
+    if (buffer_iter == arguments.end()) {
+      result->Error(
+          "Argument error",
+          "Missing argument buffer while trying to customize system cursor");
+      return;
+    }
+    auto buffer = std::get<std::vector<uint8_t>>(
+        arguments.at(flutter::EncodableValue("buffer")));
+    auto width_iter =
+        arguments.find(EncodableValue(std::string(kCustomCursorWidthKey)));
+    if (width_iter == arguments.end()) {
+      result->Error(
+          "Argument error",
+          "Missing argument width while trying to customize system cursor");
+      return;
+    }
+    auto width = std::get<int>(arguments.at(flutter::EncodableValue("width")));
```

Comment:
Could this use `width_iter->second` instead? Same comment for other arguments

Reply:
Sure